### ISSUE TITLE
Issue with bone

### ISSUE THREAD

**Issue by Gamerage** [romanvolodin/dynamic_parent#15]
File "C:\Users\Win10\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\dynamic_parent.py", line 287, in execute
    dp_create_dynamic_parent_pbone(self)
  File "C:\Users\Win10\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\dynamic_parent.py", line 172, in dp_create_dynamic_parent_pbone
    parent_obj_pbone = selected_bones[0]
                       ~~~~~~~~~~~~~~^^^
IndexError: list index out of range
Error: Python: Traceback (most recent call last):
  File "C:\Users\Win10\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\dynamic_parent.py", line 287, in execute
    dp_create_dynamic_parent_pbone(self)
  File "C:\Users\Win10\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\dynamic_parent.py", line 172, in dp_create_dynamic_parent_pbone
    parent_obj_pbone = selected_bones[0]
                       ~~~~~~~~~~~~~~^^^
IndexError: list index out of range

Not sure what it means?

**Comment by romanvolodin** [comment]
Hmmm, look's like something changed in 4.2. Thanks, will check.

**Comment by romanvolodin** [comment]
> Hmmm, look's like something changed in 4.2. Thanks, will check.

Nope, this error shows when no parent bone selected. I have to make proper message about that.

**Comment by romanvolodin** [comment]
Fixed in 334d42e77f6a420ba92a19d619d41b3620e5c0f2

**Comment by Gamerage** [comment]
Super! Ye I wasn't sure of the functionality - this added message is a good idea - thanks for the update!
